README.md

```diff
@@ -14,8 +14,8 @@ Streamlit lets you turn data scripts into sharable web apps in minutes, not week
 pip install streamlit
 streamlit hello
 ```
+Streamline can also be installed in the virtual environment of [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
```

Comment:
The project name is Streamlit, not Streamline

Reply:
Ohh yeah sorry for that.